### ISSUE TITLE
Direct Attachment without conversion into byte or passing URL

### ISSUE THREAD

**Issue by Muhammad1Nouman** [gregsdennis/Manatee.Trello#318]
I have used trello api for node, that api is also proving ability to directly attach attachments without converting file/image into bytes and without passing url? 
One of the important functionality that this code is missing is to directly attach the attachment into trello cards without converting it into bytes from desktop.

I want to open that attachment in the same tab after adding attachment.
Just like this
![f](https://user-images.githubusercontent.com/29055622/70420221-0bdb8900-1a89-11ea-9ff3-4aab8971776e.PNG)

**Comment by gregsdennis** [comment]
The API docs say to upload as a Multipart Form Data, which is what I'm doing. In .Net, this requires that I supply it with a byte array.  I'm not sure what else I could be doing here.

**Comment by gregsdennis** [comment]
I created a short test for this:

```c#
var list = _factory.List(TrelloIds.ListId);
var card = await list.Cards.Add("attachment test");

var fileData = File.ReadAllBytes(@"c:\users\gregs\desktop\images.jpg");
var attachment = await card.Attachments.Add(fileData, "test", ct);
```

It works just fine: https://trello.com/c/s3frWHY4

**Comment by Muhammad1Nouman** [comment]
After spending the whole day, I figure out the way to add attachment without converting, the file into byte array. With this feature one can view the attachment on trello board instead of downloading and then converting the byte array into image.

I made some changes in the code, one of the major change in WebApiClient File and tested the code on my trell account, It was working fine. However, when I tried to push my branch into this repository, I am getting the access denied message

![Capture](https://user-images.githubusercontent.com/29055622/71088509-6c24a600-21c0-11ea-937f-5833935fad81.PNG)
 
Here is the snippet for the some modification in WebApiClient file.

![Http](https://user-images.githubusercontent.com/29055622/71086978-f2d78400-21bc-11ea-99e9-6e46b0c6a6b3.PNG)

**Comment by gregsdennis** [comment]
> I was getting the access denied message

Yes, you have to fork the repo and create a pull request from your fork.

While those changes work for local files (specifically *only* PNG images), they may not work for other URIs or contents.

Lastly, please don't post screenshots of code; just post the code.  Text in images cannot be copied.

---

I'm not entirely sure what's not working for you with the existing code and [the code snippet I posted](https://github.com/gregsdennis/Manatee.Trello/issues/318#issuecomment-564684929).  You posted a screenshot of an open image *in Trello*, but I don't know what you're getting.  As I mentioned, it's all working fine for me and in my integration test suite which runs on Appveyor.
